graph/examples/stress.rs

```diff
@@ -0,0 +1,293 @@
+use std::alloc::{GlobalAlloc, Layout, System};
```

Comment:
I'm skipping the review for this file because I don't think it matters. Let me know if you disagree or need any particular part of it reviewed.

Reply:
No, this file is really just a tool to explore various scenarios and get a better understanding of memory usage.